nautilus_trader/backtest/node.py

```diff
@@ -161,6 +162,11 @@ def run(self) -> list[BacktestResult]:
             except Exception as e:
                 # Broad catch all prevents a single backtest run from halting
                 # the execution of the other backtests (such as a zero balance exception).
+                try:
+                    init_logging()
```

Comment:
There's also a specific function to check this as well:
```
from nautilus_trader.common.component import is_logging_initialized
```

I'll update straight to `develop` with a similar pattern using that, and credit you.

Reply:
ok, fyi, I've just made the change you mentioned

Reply:
I should have mentioned, also fixed on `develop` branch now https://github.com/nautechsystems/nautilus_trader/blob/develop/nautilus_trader/backtest/node.py#L163.

Reply:
no problem, just close the pull request please, I don't think I can do it